homeassistant/components/system_bridge/__init__.py

```diff
@@ -177,6 +193,61 @@ async def handle_open(call):
         except (BridgeAuthenticationException, *BRIDGE_CONNECTION_ERRORS) as exception:
             _LOGGER.warning("Error sending. Error was: %s", exception)
 
+    async def handle_send_keypress(call):
+        """Handle the send_keypress service call."""
+        device_registry = dr.async_get(hass)
```

Comment:
We can extract this whole part until we get the bridge to a custom voluptuous validator defined inside `async_setup_entry`. Raise `vol.Invalid` if we can't return a bridge.